### PR TITLE
updated Readme

### PR DESCRIPTION
**Pull request by richarddushime**
I have updated the README file to include instructions for setting up the project on both Windows and macOS/Linux operating systems. The revised README provides clear steps for installation and usage, ensuring that contributors/users can easily follow the instructions and set up the Student Intern Organiser application on their respective platforms.

### CODE REVIEW

**Comment by rowlandm** [comment]
Looks ok, but could you make the windows header one hash please and keep the headers simple without the extra **

**Comment by richarddushime** [comment]
> Looks ok, but could you make the windows header one hash please and keep the headers simple without the extra **

Thanks let me do it

File: README.md [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
 
 This is a way for WEHI to organise the Research Software Engineer Student Internship program
+# Installation
+To begin, please follow the steps below for installing Python and setting up the virtual environment:
 
+## Install Python
+Download the latest version of Python from the official website.
+[Download Here](https://www.python.org/)
 
+# **Windows**
```
Comment (rowlandm) [inline README.md:11]: Please remove the **

File: README.md [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
+```
+.\.env\Scripts\activate
+```
+## Run the application
+Start the Flask application by running the following command:
+```
+flask run
+```
+
+# **MacOS/Linux**
```
Comment (rowlandm) [inline README.md:37]: Please remove the **

**Comment by rowlandm** [comment]
please make the changes in this pull request :)

**Comment by richarddushime** [comment]
> please make the changes in this pull request :)

i had left the 2 bold since they are the main title in the file

**Comment by richarddushime** [comment]
Done updating the file as per suggestions

**Comment by rowlandm** [comment]
> Done updating the file as per suggestions

Thanks Richard, just one more slight change

**Comment by richarddushime** [comment]
> > Done updating the file as per suggestions
> 
> Thanks Richard, just one more slight change

Done, Thanks

**Review by rowlandm — APPROVED** [review]
Thanks Richard - good work!

**Pull request merged**